database/seed.js

```diff
@@ -1,11 +1,19 @@
-const createReview = require('./dummyData');
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
 const Review = require('./db.js').Review;
-const data = createReview();
-Review.collection.drop();
-Review.insertMany(data, (err)=> {
-  if (err) {
-    console.log("error from seed.js", err);
-  } else {
-    console.log("data is seeded");
-  }
-})
+const runScript = require('./script/seedingScript.js').runScript;
+// Connection URL
+const url = 'mongodb://localhost:27017';
+// Database Names
+const dbName = 'mashbnb1';
+// Use connect method to connect to the server
+// Drop Collection before running the seed
+MongoClient.connect(url, async function(err, client) {
+  assert.equal(null, err);
+  console.log("Connected correctly to server");
+  const db = client.db(dbName);
+  runScript(db, 10000000, 1, 250000, false);
```

Comment:
Personally I got the best performance when I ran a batch size of 100k, but if this works that's great!